src/controller/account.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const account = express.Router();
+const { User } = require('../models');
+
+account.get('/user', async (req, res) => {
+  try {
+    const { _id } = req.user;
+    const user = await User.findById(_id, '-watchlists -investments');
+    if (user) {
+      return res.send(user);
+    }
+    return res.sendStatus(404);
+  } catch (err) {
+    if (err instanceof TypeError) {
+      return res.sendStatus(401);
```

Comment:
TypeError can be thrown only in case of a wrong token? Isn't it handled by passport js?